packages/@react-spectrum/tag/test/TagGroup.test.js

```diff
@@ -299,6 +299,48 @@ describe('TagGroup', function () {
     expect(onRemoveSpy).toHaveBeenCalledWith('1');
   });
 
+  it('maxRows should limit the number of tags shown', function () {
```

Comment:
Shouldn't we test more cases? Right now I'm only thinking of one tag and three rows and there is no button.

Reply:
Is there a case where the user has max row one and a narrow width, in this case visibleTagCount is zero and showCollapseButton is true? Only the button shows, no tags show. I think so, but is that weird?

Reply:
And I think always showing at least one tag make the most sense (which is the current behavior).